os/include/tinyara/binfmt/compression/compression.h

```diff
@@ -27,6 +27,13 @@
  * Pre-processor Definitions
  ****************************************************************************/
 
+#define COMP_SUCCESS            0
+#define COMP_INVALID_TYPE       1
```

Comment:
I think we can substitute `COMP_INVALID_TYPE` to `COMP_NOT_SUPPORT`.
Because they are all for not supported type.

Reply:
changed COMP_INVALID_TYPE to COMP_NOT_SUPPORTED.